sklearn/decomposition/_lda.py

```diff
@@ -239,6 +239,10 @@ class LatentDirichletAllocation(TransformerMixin, BaseEstimator):
         after normalization:
```

Comment:
Could you modify the `components_` docstring with

```
components_ : ndarray of shape (n_components, n_features)
```

Like this, the whole list of the attribute will follow our documentation guideline.

Reply:
Done